rules/symcrypt-openssl.mk

```diff
@@ -1,6 +1,6 @@
 # SYMCRYPT_OPENSSL
 
-SYMCRYPT_OPENSSL_VERSION = 0.12
+SYMCRYPT_OPENSSL_VERSION = 1.4-preview
```

Comment:
1.4.3-preview would be more accurate. Does this override the value in the control file?

Reply:
Changed the version to 1.4.3-preview, thanks.